docs/docsite/rst/plugins/inventory.rst

```diff
@@ -11,7 +11,7 @@ Inventory plugins allow users to point at data sources to compile the inventory
 .. _enabling_inventory:
 
 Enabling Inventory Plugins
-++++++++++++++++++++++++++
+--------------------------
```

Comment:
Thanks for updating the headers @s-hertel - could you also fix the top header (line 5)? It should be underlined with =======. See https://github.com/ansible/ansible/issues/35520. Thanks!